lib/src/utils/ds_linkify.util.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:linkify/linkify.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+/// An utility class that has methods related to likified texts.
+abstract class DSLinkify {
+  static List<InlineSpan> linkifyPlainText({
```

Comment:
could use 'get' in the method name for easier understanding

Reply:
I don't think that a ```get``` prefix is suitable for this method, because ```linkify``` is an action (and an infinitive verb). With that being said, if I rename to ```getLinkify```, looks like I'm getting a link, which is not the case. So I suggest three ways to improve readability:

- Leave as ```linkify```.
- Remove this word completely, since it's already at the class name, being like ```DSLinkify.plainText```.
- Change the ```linkify``` word to a more readable verb.

What you think about that?

Reply:
really, i agree with you

in that case i think you can do the second option, remove the `linkify` from the name, because the name is in the class too